script-test/playbackstrategies/msestrategytest.js

```diff
@@ -407,9 +413,9 @@ require(
 
         it('should call plugin handler on dash baseUrl changed event', function () {
```

Comment:
Could this test name have `manifest loaded event` rather than `baseurl changed event`